app/controllers/claims/support/schools/mentors/add_mentor_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Claims::Support::Schools::Mentors::AddMentorController < Claims::ApplicationController
```

Comment:
```suggestion
class Claims::Support::Schools::Mentors::AddMentorController < Claims::Support::ApplicationController
```
Claims has a support application controller, which all support controllers need to inherit from.